specification/cognitiveservices/data-plane/Face/v1.0/Face.json

```diff
@@ -0,0 +1,2655 @@
+     "maximum": 1
+    },
+    "sideburns": {
+     "type": "number",
+     "minimum": 0,
+     "maximum": 1
+    }
+   }
+  },
+  "HeadPoseProperties": {
+   "type": "object",
+   "description": "",
+   "properties": {
+    "roll": {
+     "type": "number"
+    },
+    "yaw": {
+     "type": "number"
+    },
+    "pitch": {
+     "type": "number"
+    }
+   }
+  },
+  "EmotionProperties": {
+   "type": "object",
+   "description": "",
+   "properties": {
+    "anger": {
+     "type": "number",
+     "minimum": 0,
+     "maximum": 1
+    },
+    "contempt": {
+     "type": "number",
+     "minimum": 0,
+     "maximum": 1
+    },
+    "disgust": {
+     "type": "number",
+     "minimum": 0,
+     "maximum": 1
+    },
+    "fear": {
+     "type": "number",
+     "minimum": 0,
+     "maximum": 1
+    },
+    "happiness": {
+     "type": "number",
+     "minimum": 0,
+     "maximum": 1
+    },
+    "neutral": {
+     "type": "number",
+     "minimum": 0,
+     "maximum": 1
+    },
+    "sadness": {
+     "type": "number",
+     "minimum": 0,
+     "maximum": 1
+    },
+    "surprise": {
+     "type": "number",
+     "minimum": 0,
+     "maximum": 1
+    }
+   }
+  },
+  "FindSimilarRequest": {
+   "type": "object",
+   "required": [
+    "faceId"
+   ],
+   "description": "",
+   "properties": {
+    "faceId": {
+     "type": "string",
+     "description": "FaceId of the query face. User needs to call Face - Detect first to get a valid faceId. Note that this faceId is not persisted and will expire 24 hours after the detection call"
+    },
+    "faceListId": {
+     "type": "string",
+     "description": "An existing user-specified unique candidate face list, created in Face List - Create a Face List. Face list contains a set of persistedFaceIds which are persisted and will never expire. Parameter faceListId and faceIds should not be provided at the same time"
+    },
+    "faceIds": {
+     "type": "array",
+     "description": "An array of candidate faceIds. All of them are created by Face - Detect and the faceIds will expire 24 hours after the detection call. The number of faceIds is limited to 1000. Parameter faceListId and faceIds should not be provided at the same time.",
+     "maxLength": 1000,
+     "items": {
+      "type": "string"
+     }
+    },
+    "maxNumOfCandidatesReturned": {
+     "type": "integer",
+     "description": "The number of top similar faces returned. The valid range is [1, 1000].It defaults to 20.",
+     "default": 20,
+     "minimum": 1,
+     "maximum": 1000
+    },
+    "mode": {
+     "type": "string",
+     "description": "Similar face searching mode. It can be \"matchPerson\" or \"matchFace\". It defaults to \"matchPerson\".",
+     "default": "matchPerson",
+     "enum": [
+      "matchPerson",
+      "matchFace"
+     ]
+    }
+   }
+  },
+  "FindSimilarResponse": {
+   "type": "array",
+   "items": {
+    "$ref": "#/definitions/FindSimilarResponseElement"
+   }
+  },
+  "FindSimilarResponseElement": {
```

Comment:
I'd avoid `Response` in class names, as they affect generation and don't give a good experience.